pycytominer/tests/test_annotate.py

```diff
@@ -74,20 +103,24 @@ def test_annotate_output():
 
 
 def test_annotate_output_compress():
+    
```

Comment:
Also, why did this need to be modified? (`test_annotate_output_compress`)

The tests were passing previously

Reply:
More detailed explanation above, but in short `test_annotate_output_compress` is operating on a version of `PLATEMAP_DF` that has renamed columns and is only passing because it is being given the wrong arguments.

Reply:
As mentioned above, the use of `platemap_df = PLATEMAP_DF.copy()` is not strictly necessary for this test, but I think it maybe worth adopting that structure in all of the tests for consistency sake.